src/class.js

```diff
@@ -5,6 +5,15 @@
 // Add a method called `comparePasswords`.  `comparePasswords` should have a parameter
 // for a potential password that will be compared to the `password` property.
 // Return true if the potential password matches the `password` property.  Otherwise return false.
+class User {
+  constructor(options) {
+    this.email = options.email;
+    this.password = options.password;
+  }
+  comparePasswords(password) {
+    return (this.password === password);
```

Comment:
You don't need to wrap this line in parens.  Just do `return this.password === password;`